### PR TITLE
Starts experimental facelift branch 

### PR DESCRIPTION
**Pull request by Martskin**
Adds bourbon, neat and bitters.
Renames main.css -> main.scss.
Adds Font Awesome.
Adds normalize.scss.
Starts index page.
Starts new styles.
No JS behaviors yet.

TODO:
- [x] Style "Tech Groups" section
- [x] Style "Tech Conferences" section
- [x] Style "Coworking Spaces" section
- [x] Wire up hamburger menu on mobile
- [x] Browser test
- [ ] Trim CSS cruft
- [x] Replace greeking on welcome hero
- [x] Accommodate new "Code of Conduct" and "How to submit changes" menu items
- [x] Add active menu item script

### CODE REVIEW

**Comment by ericlathrop** [comment]
Looks pretty great, but there's some weirdness with the icon sizes in Firefox.

**Comment by ericlathrop** [comment]
I also can't make the hamburger menu do anything in FF's responsive design preview mode.

**Comment by Martskin** [comment]
Thanks! Yeah, I've only styled the home page and I haven't added jQuery/JS behaviors nor done any browser testing yet (*developing in Chrome*). Wanted to make sure you dug it before I fleshed it out. 

Shall I continue?

**Comment by ericlathrop** [comment]
Yeah, I dig it! Maybe post a screenshot in the slack channel for people too lazy to pull your branch and look at it.

**Comment by ericlathrop** [comment]
Also FYI, there's probably going to be a few more menu items like "Code of Conduct" and "How to submit changes", so keep that in mind for the navigation. I really appreciate your help!

**Comment by Martskin** [comment]
Glad to help! Okay, I'll get back into it this weekend and share on the Slack.

> few more menu items like "Code of Conduct" and "How to submit changes", so keep that in mind for the navigation. I really appreciate your help!

Cool, I'll keep that in mind.

**Comment by Martskin** [comment]
@ericlathrop I made some more progress on the facelift branch this evening. :lipstick:

**Comment by ericlathrop** [comment]
@Martskin Lookin good!
You chould replace the javascript with this to get rid of jQuery and Modernizr:
```javascript
document.querySelector("[data-behavior=\"toggle-menu\"]").addEventListener("click", function(e) {
	document.querySelector("[data-object=\"menu-object\"]").classList.toggle("is-open");
});
```
Just put it at the bottom of the page, and it will be fine.

**Comment by ericlathrop** [comment]
I'm getting really excited to get this deployed. It will be a major improvement.

**Comment by Martskin** [comment]
> document.querySelector("[data-behavior=\"toggle-menu\"]").addEventListener("click", function(e) {
>     document.querySelector("[data-object=\"menu-object\"]").classList.toggle("is-open");
> });

Nice! I like it. Okay, I removed jQuery and Modernizr. And Browser tested in IE10, IE9, Chrome, Safari and Firefox.

I also created a function to add the `is-active` modifier class to the nav items to indicate the current page. I wonder if there's a way that [unfold](https://github.com/ericlathrop/unfold) can do something like that?
image

**Comment by ericlathrop** [comment]
I'll keep that use-case in mind when working on the current unfold refactor, that will be handy.

**Comment by Martskin** [comment]
Cool. Would be useful for unique page titles as well.

**Comment by ericlathrop** [comment]
Unique titles are do-able right now, but a tad clunky.

**Comment by Martskin** [comment]
Hmm, maybe we could tap into that as a class name on the html or body tag.

File: src/css/_imports.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+/* -----------------------------------------------------------------------------
+	Imports
+*/ 
+
+@import "./node_modules/normalize.scss/normalize";
+@import "./node_modules/node-bourbon/node_modules/bourbon/app/assets/stylesheets/bourbon";
```
Comment (Martskin) [inline src/css/_imports.scss]: @sean-clayton I'm sure there's a more elegant way to import Bourbon and Neat here but I couldn't get it to compile.

Comment (sean-clayton) [inline src/css/_imports.scss]: Actually, as far as I know, there isn't at the moment. I've been trying to think of a way to come up with a solution for @ericlathrop's Unfold. It uses a JSON file and I was thinking of adding something like `"includePaths": ["path1","path2"]` to it but for bourbon it's not a string, it's `require('node-bourbon')` as the path which doesn't really work in JSON.

As far as putting Bourbon into the project, that's as clean as it gets for the moment :+1:

Comment (ericlathrop) [inline src/css/_imports.scss]: I used bourbon with unfold on twoscoopgames.com, but I didn't load it through NPM, just dropped the folder inside of the css folder. See here:
https://github.com/TwoScoopGames/twoscoopgames.com/tree/master/site/css

@sean-clayton I'm in the middle of a big unfold refactor, so I'll try to get this include path thing working when I get to the SASS part of it.

**Comment by Martskin** [comment]
I added the Louisville.io logo and favicon we were discussing in Slack to the facelift branch. Lemme know if you want me to keep working on this branch or not.

**Comment by ericlathrop** [comment]
I think it looks pretty great. What do you think about adding in the cards from Austin's screenshot? I like them.

You should probably merge in the changes from master since there's been some work done.

Also, what do you think about deploying this version as it is right now, and patching in more work in a separate PR?

**Comment by Martskin** [comment]
Awesome! Okay, I made those changes and removed the greeking text on the home page. One thing to note before you deploy is that we had added the `Code of Conduct` and `How to Submit Changes` links to the hero and footer that currently 404.

**Comment by Martskin** [comment]
Oh, did you want the blue/red/green border off the cards too?

**Comment by ericlathrop** [comment]
Yes please. Also, I think you merged your own master, not louisvilleio/master.

**Comment by Martskin** [comment]
Okay done.

And I had to merger in master from `louisville.io` to my fork since that's where this PR originated. I'm pretty sure it got everything new.

https://github.com/Martskin/louisville.io/commit/f543f7886712040eb921f7c693cbfb85df5ea0a7

**Pull request merged**